src/xdocs/checks/annotation/annotationusestyle.xml

```diff
@@ -130,116 +130,136 @@
         </div>
       </subsection>
       <subsection name="Examples" id="Examples">
-        <p> To configure the check:</p>
+        <p id="Example1-config">
+          To configure the check:
+        </p>
         <source>
-&lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>
-          Example:
-        </p>
+        <p id="Example1-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;unchecked&quot;) // OK
-@Deprecated // OK
-@SomeArrays({&quot;unchecked&quot;,&quot;unused&quot;}) // OK
-public class TestOne
+@SuppressWarnings(&quot;unchecked&quot;) // ok as it's in implied style
+@Deprecated // // ok as it matches closingParens default property
+@SomeArraysUnique({&quot;unchecked&quot;,&quot;unused&quot;}) // ok as it's in implied style
+public class Example1
 {
 
 }
 
-@SuppressWarnings(value={&quot;unchecked&quot;}) // Violation - parameter 'value' shouldn't be used
```

Comment:
Please restore `@SuppressWarnings` that we lost over second class.

Reply:
done